pkg/kafkautil/client.go

```diff
@@ -73,13 +81,11 @@ func New(opts *KafkaConfig) (client KafkaClient, err error) {
 	}
 
 	if kclient.brokers, err = kclient.DescribeCluster(); err != nil {
-		kclient.admin.Close()
```

Comment:
I think this is necessary still for garbage collection. If it got to the point where it is trying to `DescribeCluster`, it has already successfully opened an admin client. If that errs out then we want to make sure we close it I think.

Reply:
Thanks, I just added this line